indico/queries/submission.py

```diff
@@ -328,16 +328,31 @@ class SubmitReview(GraphQLRequest):
 
     Options:
         changes (dict or JSONString): changes to make to raw predictions
+
         rejected (boolean): reject the predictions and place the submission
             in the review queue. Must be True if $changes not provided
 
+        force_complete (boolean): have this submission bypass the Review queue
+            (or exceptions queue if <rejected> is True) and mark as Complete.
+            NOT RECOMMENDED
+
     Returns:
         Job: A Job that can be watched for status of review
     """
 
     query = """
-        mutation SubmitReview($submissionId: Int!, $changes: JSONString, $rejected: Boolean) {
-            submitAutoReview(submissionId: $submissionId, changes: $changes, rejected: $rejected) {
+        mutation SubmitReview(
+            $submissionId: Int!,
+            $changes: JSONString,
+            $rejected: Boolean,
+            $forceComplete: Boolean
+        ) {
+            submitAutoReview(
+                submissionId: $submissionId,
+                changes: $changes,
+                rejected: $rejected,
+                forceComplete: $forceComplete
```

Comment:
we need to move some of these to add them to the query if set otherwise we needlessly break backwards compatibility. I presume the default is false and isn't a required arg?

Reply:
yes, not a required arg - so shouldn't break backwards compat...

Reply:
oh but with fog - hmmm, I see... yeah, not sure - I think this was part of the convo around platform/client versioning and release notes. what do you think here? try with the kwarg and handle "invalid query" errors by removing some kwargs?

Reply:
oh cool! alright - will switch to using this :+1: